common-util/src/main/java/org/sunbird/common/responsecode/ResponseMessage.java

```diff
@@ -317,6 +317,27 @@
     String CASSANDRA_CONNECTION_ESTABLISHMENT_FAILED =
         "Cassandra connection establishment failed in {0} mode.";
     String COMMON_ATTRIBUTE_MISMATCH = "{0} mismatch of {1} and {2}";
+    String JSON_TRANSFORM_INVALID_TYPE_CONFIG =
+        "Invalid type configuration for JSON transformation";
+    String JSON_TRANSFORM_INVALID_DATE_FORMAT =
+        "Invalid date format configuration for JSON transformation";
+    String JSON_TRANSFORM_FAILED_INVALID_INPUT =
+        "JSON Transformation failed due to invalid input data format";
```

Comment:
ERROR_JSON_TRANSFORM_INVALID_INPUT
JSON transformation failed as invalid input provided for field {0}.